docs: update readme information to indicate that some features could fail since the Open edX Olive (#52)
eduNEXT/flow-control-xblock: README.rst

* Enter the problem locator ids (as many as required) to evaluate the condition.
* Select an action to apply when the condition is met.

> ⚠️ Since Open edX Olive some the features could fail because to changes in MFE integration UI.

Reviewer: ```suggestion
⚠️ Since the Open edX Olive release, some features could fail because of the integration of MFE.
```

And I suggest adding this message before the installation instructions. To make it quickly visible.

Reviewer: Also, if the only MFE that causes problems is learning MFE, it is better if we are specific in that.
